lib/config.js

```diff
@@ -19,6 +19,24 @@ const BrunchError = require('./error');
 const normalizeChecker = require('anymatch');
 const defaultConfigFilename = 'brunch-config';
 
+const sanitizeConfig = config => {
```

Comment:
Maybe we can come up with less general name than `sanitizeConfig`? `trimTrailingSlashes`?